tests/unit/tests_high_five_base.cpp

```diff
@@ -78,7 +78,7 @@ struct ContentGenerate {
 
     T operator()() {
         T ret = _init;
-        _init += _inc;
+        _init = static_cast<T>(_init + _inc);
```

Comment:
Sorry a last remark here. The compiler used to warn on an assignment but doesn't warn if we sum? When we sum it should anyway use the longest type and so the cast is not needed.

Reply:
gcc emits a warning in such case because as addition might overflow, the compiler stores addition result into an int, which then fails when trying to put it in `k`:

```
cat <<EOF | g++ -Wconversion -x c++ -
int main() {
        unsigned char i = 0, j = 0;
        unsigned char k = i + j;
        return k;
}
EOF
<stdin>: In function ‘int main()’:
<stdin>:3:22: warning: conversion to ‘unsigned char’ from ‘int’ may alter its value [-Wconversion]
```